sda-commons-server-auth/README.md

```diff
@@ -306,3 +306,47 @@ opa:
 
 [`sda-commons-server-auth-testing`](../sda-commons-server-auth-testing/README.md) provides support for testing
 applications with authentication.
+
+## Input Extensions
+
+The Bundle offers the option to register extensions that send custom data to the Open Policy Agent to be accessed during policy execution.
+Such extensions implement the [`OpaInputExtension`](.src/main/java/org/sdase/commons/server/opa/extension/OpaInputExtension.java) interface and are registered in a dedicated namespace.
+The extension is called in the `OpaAuthFilter` and is able to access the current `RequestContext` to for example extract additional data from the request.
+_Overriding existing input properties (`path`, `jwt`, `httpMethod`) is not possible._
+
+This extension option should only be used if the normal authorization via constraints is not powerful enough.
+In general, a custom extension should not be necessary for most use cases.
+
+_Remark on accessing the request body in an input extension:_
+The extension gets the `ContainerRequestContext` as input to access information about the request.
+Please be aware to not access the request entity since this might break your service.
+A test that shows the erroneous behavior can be found in [`OpaBundleBodyInputExtensionTest.java`](./src/test/java/org/sdase/commons/server/opa/OpaBundleBodyInputExtensionTest.java)
```

Comment:
👍 

Should we add a security note here that the service might only consider one value of a specific header or query param while the OPA is able to authorize on an array of those?